src/etc/inc/shaper.inc

```diff
@@ -4095,6 +4507,16 @@ class dnqueue_class extends dummynet_class {
 
 	function validate_input($data, &$input_errors) {
 		parent::validate_input($data, $input_errors);
+		
+		
+		/* Limiter patch */
+		$selectedAqm = getAQMs()[$data['aqm']];
+		if (!$selectedAqm)
+			$input_errors[] = gettext("Selected scheduler not recognized: " . $data['sched'] . '.');
```

Comment:
See previous comment about printing this value, and also the message says scheduler but it looks like it should be AQM